src/test/java/com/meilisearch/integration/SettingsTest.java

```diff
@@ -176,18 +201,19 @@ public void testResetRankingRuleSettings() throws Exception {
         index.waitForTask(index.updateRankingRuleSettings(newRankingRules).getUid());
         String[] updatedRankingRuleSettings = index.getRankingRuleSettings();
 
+        index.waitForTask(index.resetRankingRuleSettings().getUid());
```

Comment:
Why this had to change?

Reply:
To group the test steps as you suggest, I moved this one line in the `setup` steps so that all `asserts` are grouped afterward in the `verify` steps